### PR TITLE
Update format-text-paragraphs-with-textwrap.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/python-core/string-recipes/format-text-paragraphs-with-textwrap.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/string-recipes/format-text-paragraphs-with-textwrap.md?ref=77745b3ded884fb78da308d323e17dc078cb00c0)

File: python/python-core/string-recipes/format-text-paragraphs-with-textwrap.md [hunk 1]
```diff
@@ -51,27 +50,23 @@ textwrap.wrap("parallel", width=4)
...
+If we would like to include trailing newlines (`\n`) after each string of a certain `width` we can use the following syntax:
 
 ```python
 '\n'.join(textwrap.wrap('text', width=2))
 # Output:
 # 'te\nxt'
 ```
 
-Or we can use the `fill` method implemented in `textwrap` module:
+The `fill` method is shorthand notation for the above example:
```
Comment (kapnobatai137) [inline python/python-core/string-recipes/format-text-paragraphs-with-textwrap.md:61]:
```suggestion
You can also use `fill` (a shorthand notation) to achieve the same result:
```

File: python/python-core/string-recipes/format-text-paragraphs-with-textwrap.md [hunk 2]
```diff
@@ -51,27 +50,23 @@ textwrap.wrap("parallel", width=4)
...
 # Output:
 # 'te\nxt'
 ```
 
-Fill a text:
-
-```python
-textwrap.fill(text, width=55)
-```
+> 💡 `fill` doesn't require explicitly writing `\n`, it automatically creates a new line after each specified `width`.
```
Comment (kapnobatai137) [inline python/python-core/string-recipes/format-text-paragraphs-with-textwrap.md:69]:
```suggestion
> 💡 `fill` accepts the same keyword arguments as `wrap`
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**